resources/zib/zib-PharmaceuticalProduct.xml

```diff
@@ -0,0 +1,176 @@
+      <alias value="ProductCode" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.12--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-pharmaceuticalproduct-v2.1.2-2020EN" />
+        <map value="NL-CM:9.7.19927" />
+        <comment value="MedicationCode" />
+      </mapping>
+    </element>
+    <element id="Medication.code.text">
+      <path value="Medication.code.text" />
+      <short value="Medication" />
+      <definition value="There is no code for medication entered in free text. In these cases, enter the complete description." />
+      <alias value="ProductNaam" />
+      <mapping>
+        <identity value="zib-pharmaceuticalproduct-v2.1.2-2020EN" />
+        <map value="NL-CM:9.7.19929" />
+        <comment value="Medication" />
+      </mapping>
+    </element>
+    <element id="Medication.form">
+      <path value="Medication.form" />
+      <short value="PharmaceuticalForm" />
+      <definition value="The pharmaceutical form indicates the form of the medication. Examples include: tablet, suppository, infusion liquid, ointment. If the product has a GPK code in the G standard, the form will be known in the G standard. For products without a code (free text, preparation by the pharmacy), the means of administration can be entered." />
+      <alias value="FarmaceutischeVorm" />
+      <binding>
+        <strength value="example" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.9.7.8--20200901000000" />

```

Comment:
What is the reason that we choose the binding strength of FHIR instead of the zib?

FHIR = example
zib = required 

https://hl7.org/fhir/r4/medication.html

Reply:
@Ashwin-nictiz ; good catch. Fixed! Thnx.